### PR TITLE
Add mariadb-image to prow & Jenkins

### PR DESCRIPTION
**Pull request by dtantsur**

### CODE REVIEW

File: jenkins/scripts/files/run_integration_tests.sh [hunk 1]
```diff
@@ -109,6 +109,10 @@ then
   export SUSHY_TOOLS_LOCAL_IMAGE="${IRONIC_LOCAL_IMAGE}/resources/sushy-tools"
   export VBMC_LOCAL_IMAGE="${IRONIC_LOCAL_IMAGE}/resources/vbmc"
 
+elif [ "${REPO_NAME}" == "mariadb-image" ]
+then
+  export MARIADB_LOCAL_IMAGE="/home/${USER}/tested_repo"
+
```
Comment (fmuyassarov) [inline jenkins/scripts/files/run_integration_tests.sh:115]: That's Jenkins configuration. Are you expecting to run integration tests for pull requests ?

Comment (dtantsur) [reply jenkins/scripts/files/run_integration_tests.sh:115]: I hope so, yes. Why?

Comment (fmuyassarov) [reply jenkins/scripts/files/run_integration_tests.sh:115]: Because title says to Prow, so I was expecting only Prow addition :)

Comment (dtantsur) [reply jenkins/scripts/files/run_integration_tests.sh:115]: Ah, hmm. I guess I'm not strong in terminology :) I want it to run the absolutely normal CI.

**Review by fmuyassarov — COMMENTED** [review]
/approve
/retitle Add mariadb-image to prow & Jenkins

**Review by elfosardo — APPROVED** [review]

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**